### ISSUE TITLE
Error while using GenCase

### ISSUE THREAD

**Issue by leafnlight** [DualSPHysics/DesignSPHysics#88]
Hi, I'm trying to reproduce the sloshingTank example (https://www.youtube.com/watch?v=x2u6UIicIEY) and I'm getting an error while running GenCase. I'm pretty sure my version is up to date since I've installed FreeCAD and the addon today.

Here's the error I get :

Traceback (most recent call last):
03:15:23    File "C:\Users\Lucas Aubé\AppData\Roaming\FreeCAD\Mod\DesignSPHysics\mod\widgets\dock\dock_pre_processing_widget.py", line 240, in on_execute_gencase
    total_particles_text = output[output.index("Total particles: "):output.index(" (bound=")]
03:15:23  ValueError: substring not found
03:15:23  

Any idea on what could be causing this?

**Comment by leafnlight** [comment]
This might help solve the issue: I've made it work by converting into comments lines 240 to 243 of dock_pre_processing_widget.py. With this modification, it works, but the simulation pop-up window doesn't display the number of particles. Which isn't a huge issue for this particular study I am doing.

**Comment by zlotny** [comment]
Hi there! Thanks for reporting that issue. It´s a problem when parsing the output given by GenCase, the latest GenCase version must have changed something on how it prints the total number of particles in the stdout. 

I´ll give it a look.

**Comment by Muyii3** [comment]
Hi, I also encountered this problem. When I follow DualSPHysics How to set up and run: CaseWaves, the error message is as follows. Is there a solution now? I actually tried to comment out the error line as well, but that didn't work, there were more errors.

02:36:27  Traceback (most recent call last):
02:38:48    File "C:\Users\Liu\AppData\Roaming\FreeCAD\Mod\DesignSPHysics\mod\widgets\dock\dock_pre_processing_widget.py", line 255, in on_execute_gencase
    total_particles_text = output[output.index("Total particles: "):output.index(" (bound=")]
02:38:48  ValueError: substring not found
